src/app/locale/en_US/Zendesk_Zendesk.yml

```diff
@@ -224,7 +224,7 @@ parts:
   - translation:
         key: "text.integrations.magento.config.test_connection.tips"
         title: "Configuration message: suggestion to look at provided URL for troubleshooting"
-        value: 'Troubleshooting tips can be found at <a href="%s" target="_blank">%s</a>'
```

Comment:
When you need to modify the text,  please create a new key and mark this as `obsolete`:

https://zendesk.atlassian.net/wiki/display/I18N/Editing+strings